### PR TITLE
Login Page with DB verification

### PR DESCRIPTION
**Pull request by surabhi-raghavan**

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -26,23 +25,23 @@ const NAVIGATION: Navigation = [
     icon: <DashboardIcon />,
   },
   {
-    segment: 'orders',
-    title: 'Orders',
-    icon: <ShoppingCartIcon />,
+    segment: 'listings',
+    title: 'Your Listing',
```
Comment (parushapradhan) [inline src/pages/_app.tsx:29]: You need to hide this if. your user isn't logged in

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -26,23 +25,23 @@ const NAVIGATION: Navigation = [
...
+  const [excludeLayout, setExcludeLayout] = React.useState(false);
 
+  // Detect if the current route should exclude the layout (e.g., /login, /register)
   React.useEffect(() => {
+    setIsClient(true);
+    setExcludeLayout(['/login', '/register'].includes(router.pathname));
+
     const token = Cookies.get('auth_token');
     if (token) {
       fetch('/api/me', {
```
Comment (parushapradhan) [inline src/pages/_app.tsx:47]: please change name from me to get user session. makes it easier to understand

File: src/pages/_app.tsx (hunk 1 above)
Comment (surabhi-raghavan) [reply src/pages/_app.tsx:29]: The your listing page also needs to be hidden if not logged in?

File: src/pages/_app.tsx (hunk 2 above)
Comment (surabhi-raghavan) [reply src/pages/_app.tsx:47]: this is done
will push the change in 10 mins

**Comment by surabhi-raghavan** [comment]
@parushapradhan  Have made the changes- check and merge branches!

**Pull request merged**